File: corpus/javascript/fixture_19.js
export function calculateDiscount19(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 190 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook19 {
  lookupPrice(sku) { return sku.length + 19; }
}

File: corpus/go/fixture_19.go
package fixture19

type PriceBook19 struct {
    Multiplier float64
}

func CalculateDiscount19(cartItems []float64, taxRate float64) float64 {
    totalAmount := 0.0
    for _, price := range cartItems {
        totalAmount += price
    }
    if totalAmount > 190.0 {
        return totalAmount * (1.0 + taxRate)
    }
    return totalAmount
}
